osu.Game/Screens/Footer/ScreenBackButton.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Localisation;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Screens.Footer
+{
+    public partial class ScreenBackButton : ShearedButton
+    {
+        // todo: see https://github.com/ppy/osu-framework/issues/3271
+        private const float torus_scale_factor = 1.2f;
+
+        public const float BUTTON_WIDTH = 240;
+
+        public ScreenBackButton()
+            : base(BUTTON_WIDTH, 70)
+        {
+        }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            ButtonContent.Child = new FillFlowContainer
+            {
+                X = -10f,
+                RelativeSizeAxes = Axes.Both,
+                Direction = FillDirection.Horizontal,
+                Spacing = new Vector2(20f, 0f),
+                Children = new Drawable[]
+                {
+                    new SpriteIcon
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        Size = new Vector2(20f),
+                        Icon = FontAwesome.Solid.ChevronLeft,
+                    },
+                    new OsuSpriteText
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        Font = OsuFont.TorusAlternate.With(size: 20 * torus_scale_factor),
+                        Text = CommonStrings.Back,
+                        UseFullGlyphHeight = false,
+                    }
+                }
+            };
+
+            DarkerColour = Color4Extensions.FromHex("#DE31AE");
+            LighterColour = Color4Extensions.FromHex("#FF86DD");
```

Comment:
I'm not 100% on these colours but will amend once things come together if they don't work.